### ISSUE TITLE
lab ci trace should be able to trace all jobs in a pipeline

### ISSUE THREAD

**Issue by varac** [zaquestion/lab#344]
I'd like to watch the output of all jobs of a running pipeline, so I don't need to run `lab ci trace` again and again once a job finished.

**Comment by zaquestion** [comment]
That seems like it could get pretty messy, did you have a way in mind that this might work? Perhaps like tail with multiple files.

**Comment by varac** [comment]
@zaquestion Yes, I know it could get messy, but it's better than not being able to have this option at all. Indeed, it could work like tailing mutiple files, and I propose to prepend the job name in front of every line output in this case.

**Comment by bmeneguele** [comment]
IMHO, "it's better than not being able to have this option at all" isn't true.
If you have 2 or 3 jobs running simultaneously, ok, it might work, but what happens when you have 6 or 7 jobs running at the same time like I do? The implementation of this feature won't be easy to achieve and is really error-prone.

What do you think about using `watch "lab ci trace"` in different tmux/screen windows? Like I do when needed.
